0.3.7 cannot be used as dependence

My program, that relies on `curl ^0.3`, cannot be built because of the following error:
```
error[E0425]: unresolved name `curl_sys::CURL_ERROR_SIZE`
    --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/curl-0.3.7/src/easy.rs:3132:50
     |
3132 |     handle.data.error_buf = RefCell::new(vec![0; curl_sys::CURL_ERROR_SIZE]);
     |                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^
~/.cargo/registry/src/github.com-1ecc6299db9ec823/curl-0.3.7/src/easy.rs:3132:42: 3132:76 note: in this expansion of vec! (defined in <std macros>)
```

It seesm that `curl_sys::CURL_ERROR_SIZE` has been introduced in `curl-sys 0.2.3` but curl's `Cargo.toml` uses `0.2.2`?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,7 @@ description = "Rust bindings to libcurl for making HTTP requests"
 
 [dependencies]
 libc = "0.2"
-curl-sys = { path = "curl-sys", version = "0.2.2" }
+curl-sys = { path = "curl-sys", version = "0.2.3" }
 
 # Unix platforms use OpenSSL for now to provide SSL functionality
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies]

